blockchain/chainio.go

```diff
@@ -920,7 +920,6 @@ func deserializeUtxoEntry(serialized []byte) (*UtxoEntry, error) {
 		stakeExtra := make([]byte, len(serialized[offset:]))
 		copy(stakeExtra, serialized[offset:])
 		entry.stakeExtra = stakeExtra
-		offset += len(serialized[offset:])
```

Comment:
I'd prefer to leave this.  Even though it's technically not used, if any fields are added, removing this would cause the offset to be incorrect if it weren't added back, and it's pretty likely it would be missed.